emails/templates/emails/wrapped_email.html

```diff
@@ -123,20 +123,18 @@
           <img width="30" src="{{ SITE_ORIGIN }}/static/images/email-images/relay-icon.png" style="margin-right: 5px; display: inline-block; vertical-align: top;" alt="relay icon"/>   
            
           <p style="margin-top: 0; margin-bottom: 5px; display: inline-block;"> 
-            {% with display_email|striptags|urlencode as mask_url %}
             <span class="forwarded-from-email" style="display: block; color: #FFFFFF;"> 
-            {% ftlmsg 'relay-email-forwarded-from' %} 
-            <a class="container-link" href="{{ SITE_ORIGIN }}/accounts/profile/#{{ mask_url }}" style="margin-right: 30px;color: #FFFFFF;"> {{ display_email|striptags }} </a>
+            {% with display_email|striptags|urlencode as mask_url %}
+            {% ftlmsg 'relay-email-forwarded-from-html' url=SITE_ORIGIN|add:'/accounts/profile/#'|add:mask_url attrs='class="container-link" style="margin-right: 30px;color: #FFFFFF;"' email_address=display_email|striptags %}
             {% endwith %}
-            </span> 
+            </span>
 
-          <span style="margin-top: 0; margin-bottom: 5px; display: block;color: #FFFFFF;">{% ftlmsg 'relay-email-by-line' %} <a class="container-link" href="{{ SITE_ORIGIN }}/accounts/profile" style="margin-right: 30px;color: #FFFFFF;">
-            {% if has_premium %} 
-              {% ftlmsg 'relay-email-premium-by-line-link' %} 
+          <span style="margin-top: 0; margin-bottom: 5px; display: block;color: #FFFFFF;">
+            {% if has_premium %}
+              {% ftlmsg 'relay-email-premium-byline-html' url=SITE_ORIGIN|add:'/accounts/profile/' attrs='class="container-link" style="margin-right: 30px;color: #FFFFFF;"' %}
```

Comment:
change (blocking): same change here - don't change the template until the message is translated.